src/smokeTest/java/uk/gov/hmcts/reform/jobscheduler/controllers/JobsControllerSmokeTest.java

```diff
@@ -3,27 +3,22 @@
 import io.restassured.RestAssured;
 import org.junit.Before;
 import org.junit.Test;
-import org.springframework.core.io.ClassPathResource;
-import org.springframework.core.io.Resource;
-import org.springframework.core.io.support.PropertiesLoaderUtils;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.http.HttpHeaders;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
 
-import java.io.IOException;
-import java.util.Map;
-import java.util.Properties;
-
+@RunWith(SpringRunner.class)
+@TestPropertySource("/smoke-application.properties")
```

Comment:
could just rename the properties file to `application.properties`. not sure why `smoke-` was prefixed here